src/components/Book.js

```diff
@@ -1,19 +1,32 @@
 import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { mapDispatchToProps } from "../reducers";
 
-const Book = ({id, title, category}) => (
-        <tr>
-            <td>{id}</td>
-            <td>{title}</td>
-            <td>{category}</td>
-        </tr>
-    )
+const Book = ({id, title, category, removeBook}) => {
+        const handleRemoveBook = () => {
+            removeBook({id, title, category})
+        }   
+        return (
+            <tr>
+                <td>{id}</td>
+                <td>{title}</td>
+                <td>{category}</td>
+                <td>
+                    <button onClick={handleRemoveBook} name='removeBook' type="button" value='Remove book'>
```

Comment:
- [ ] [REQUIRED] Instead of attaching `handleRemoveBook` that you have implemented in this component, you should attach passed `handleRemoveBook` from `BookList` component to on click event of `Remove book` button.